src/core/public/notifications/notifications_service.ts

```diff
@@ -56,7 +61,18 @@ export class NotificationsService {
       },
     });
 
-    return { toasts: this.toasts.setup({ i18n }) };
+    const notificationSetup = { toasts: this.toasts.setup({ i18n: i18nDep }) };
+
+    this.uiSettingsErrorSubscription = uiSettings.getUpdateErrors$().subscribe(error => {
+      notificationSetup.toasts.addDanger({
+        title: i18n.translate('core.uiSettings.unableUpdateUISettingNotificationMessageTitle', {
```

Comment:
shouldn't key be `core.notifications.unableUpdateUISettingNotificationMessageTitle`?